zoomage_report.CURATED.tsv

```diff
@@ -56187,3 +56187,215 @@ strain	mixed C57BL/6J and SV129	http://www.ebi.ac.uk/efo/EFO_0000601	E-TABM-426
+inferred cell type - authors labels	CD4+ conventional T cell	http://purl.obolibrary.org/obo/CL_0000624	E-HCAD-1	HCATisStab7463847_S1_L001-AGACGTTGTCCGTTAA	CD4-positive, alpha-beta T cell	mismatch
+inferred cell type - authors labels	CD4+ naive T cell	http://purl.obolibrary.org/obo/CL_0000895	E-HCAD-4	group3-CCGGTAGGTTCAGCGC	naive thymus-derived CD4-positive, alpha-beta T cell|naive CD4+ T Cells	mismatch
+inferred cell type - authors labels	CD8+ cytotoxic T cell	http://purl.obolibrary.org/obo/CL_0000794	E-HCAD-1	HCATisStab7747198_S1_L001-CGCCAAGGTCGCGAAA	CD8-positive, alpha-beta cytotoxic T cell	mismatch
+inferred cell type - authors labels	central venous liver sinusoidal endothelial cell	http://www.ebi.ac.uk/efo/EFO_0010706	E-HCAD-9	group4-CGAGCACTCACTCCTG	central venous hepatic sinusoidal endothelial cell	mismatch
+inferred cell type - authors labels	columella/ root cap	http://purl.obolibrary.org/obo/PO_0020130	E-GEOD-158761	SAMN16293296-AAGGAGCAGTGGGATC	columella root cap cell	mismatch
+inferred cell type - authors labels	columella root cap initial cell	http://purl.obolibrary.org/obo/PO_0020133	E-GEOD-141730	SAMN13520724-TGGGAAGCAAGTCTGT	columella root cap initial cell	mismatch
+inferred cell type - authors labels	companion cell	http://purl.obolibrary.org/obo/PO_0000071	E-GEOD-141730	SAMN13520724-GACTGCGCAAGCGTAG	companion cell	mismatch
+inferred cell type - authors labels	cytotoxic CD8+ T cell	http://purl.obolibrary.org/obo/CL_0000794	E-HCAD-1	HCATisStab7587203_S1_L001-TGGTTAGGTAAACGCG	CD8-positive, alpha-beta cytotoxic T cell	mismatch
+inferred cell type - authors labels	dendritic progenitor	http://purl.obolibrary.org/obo/CL_0001029	E-HCAD-6	group6-CAGAATCTCGGAGCAA	common dendritic progenitor	mismatch
+inferred cell type - authors labels	embryo sac central cell	http://purl.obolibrary.org/obo/PO_0020090	E-ENAD-50	SRR7812263	embryo sac central cell	mismatch
+inferred cell type - authors labels	embryo sac egg cell	http://purl.obolibrary.org/obo/PO_0025123	E-ENAD-50	SRR7812287	embryo sac egg cell	mismatch
+inferred cell type - authors labels	epidermal initial cell	http://purl.obolibrary.org/obo/PO_0000349	E-GEOD-141730	SAMN13520734-GCAATCACAATTCCTT	epidermal initial cell	mismatch
+inferred cell type - authors labels	granulocyte progenitor	http://purl.obolibrary.org/obo/CL_0000557	E-MTAB-9067	ERR4148034	granulocyte monocyte progenitor cell	mismatch
+inferred cell type - authors labels	juxtaglomerular complex cell	http://purl.obolibrary.org/obo/CL_1000618	E-GEOD-124472	SRR8377704-TTCGAAGTCATCGCTC	juxtaglomerular complex cell	mismatch
+inferred cell type - authors labels	lateral root primordia	http://purl.obolibrary.org/obo/PO_0000016	E-GEOD-158761	SAMN16293294-ATGTGTGGTTTACTCT	lateral root primordium	mismatch
+inferred cell type - authors labels	lymph node endothelial cell	http://purl.obolibrary.org/obo/CL_0002138	E-EHCA-2	22028_2_203	endothelial cell of lymphatic vessel	mismatch
```

Comment:
Lymph node isn't quite the same as lymphatic vessel. So probably wouldn't map that.